### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Geksus**

### CODE REVIEW

**Review by Oleksandr563487953 — APPROVED** [review]

File: tests/test_main.py [hunk 1]
```diff
@@ -1 +1,19 @@
...
+
+    assert cryptocurrency_action(102) == "Do nothing"
+    assert cryptocurrency_action(104) == "Do nothing"
+    assert cryptocurrency_action(96) == "Do nothing"
+    assert cryptocurrency_action(118) == "Sell all your cryptocurrency"
+    assert cryptocurrency_action(143) == "Sell all your cryptocurrency"
+    assert cryptocurrency_action(123) == "Sell all your cryptocurrency"
+    assert cryptocurrency_action(90) == "Buy more cryptocurrency"
+    assert cryptocurrency_action(75) == "Buy more cryptocurrency"
+    assert cryptocurrency_action(23) == "Buy more cryptocurrency"
```
Comment (MasterpieceElbow) [inline tests/test_main.py:19]: Add boundary conditions, you don't check when get_exchange_rate_prediction return value equals exactly 1.05 or 0.95 * value passed in cryptocurrency_action.

Also group your tests by equivalence classes.

**Review by MasterpieceElbow — CHANGES_REQUESTED** [review]
